tests/tests_unit/tests_db_connection_unit/test_clickhouse_unit.py

```diff
@@ -75,7 +94,7 @@ def test_clickhouse_get_packages(package_version, apache_http_client_version, ex
         ("a.b.c", "5.3.1"),
     ],
 )
-def test_invalid_versions_raise_error(package_version, apache_http_client_version):
+def test_clickhouse_invalid_version(package_version, apache_http_client_version):
```

Comment:
```suggestion
def test_clickhouse_get_packages_invalid_version(package_version, apache_http_client_version):
```
А то можно подумать, мы проверяем версию самого Clickhouse